### ISSUE TITLE
RSI/CCI predictions

### ISSUE THREAD

**Issue by quantum8** [CkCreative/binance-spot-trading-bot#8]
Basic RSI/CCI divergence detection could be implemented to help predict bullish/bearish divergence signals and thus set some appropriate buy/sell limits using this info.

**Comment by CkCreative** [comment]
Good point. This is part of the important features in the next major update when I find time. Thanks!

**Comment by quantum8** [comment]
Awesome. At least it's now logged :)

For something simple that might align with this style of bot could be the use of the supertrend indicator in conjunction with the RSI/CCI indicators.
